Lambdas/GHHooks/EventHandler/Handlers/DocsIssuer.swift

```diff
@@ -0,0 +1,135 @@
+import GitHubAPI
+import DiscordBM
+import Logging
+
+/// Sends a "Need translation" message for each PR in a push-commit that needs that.
+struct DocsIssuer {
+
+    enum Configuration {
+        static let docsRepoID = 64560805
+    }
+
+    let context: HandlerContext
+    let commitSHA: String
+    let repo: Repository
+    var logger: Logger {
+        context.logger
+    }
+
+    init(context: HandlerContext) throws {
+        self.context = context
+        self.commitSHA = try context.event.after.requireValue()
+        self.repo = try context.event.repository.requireValue()
+    }
+
+    func handle() async throws {
+        guard repo.id == Configuration.docsRepoID else {
+            return
+        }
+        for pr in try await getPRsRelatedToCommit() {
+            guard isNotExemptFromNewIssue(pr: pr) else {
+                logger.warning(
+                    "Will not file issue for docs push PR because it's exempt from new issues",
+                    metadata: ["number": .stringConvertible(pr.number)]
+                )
+                continue
+            }
+            let files = try await getPRFiles(number: pr.number)
+            /// PR must contain file changes for files that are in the `docs` directory.
+            /// Otherwise there is nothing to be translated and there is no need for a new issue.
+            guard files.contains(where: { file in
+                file.filename.hasPrefix("docs/") &&
+                [.added, .modified].contains(file.status)
+            }) else {
+                logger.warning(
+                    "Will not file issue for docs push PR because no docs files are added or modified",
+                    metadata: ["number": .stringConvertible(pr.number)]
+                )
+                continue
+            }
+            try await fileIssue(number: pr.number)
+        }
+    }
+
+    /// Should not contain any labels that indicate no need for a new issue.
+    func isNotExemptFromNewIssue(pr: SimplePullRequest) -> Bool {
```

Comment:
This method name requires parsing a double-negative, which is both not idiomatic Swift and is just generally awkward. It's better to have to write `guard !isExemptFromNewIssue(...)` above than to do the extra mental gymnastics to make sense of the triple negative the guard introduces as things stand.